### DISCUSSION TITLE
JS file in root path

### DISCUSSION THREAD

**Discussion by jehoshua7** [razonyang/hugo-theme-bootstrap#227] [Q&A] [answered]
I'm using the latest release - https://github.com/razonyang/hugo-theme-bootstrap/releases/tag/v0.45.0 , and when I deployed there was a JS file in the root of ..../public

It's called "bootstrap.min.b5d86dd3a5f60c90be38a252bb65fc1a2732f32e71dc12c051720f0c7aef3cde.js"

It's no big deal, but it probably can go in the /js folder ??

**Comment by razonyang** [comment, marked as answer]
Fixed by 99b4536

**Comment by jehoshua7** [comment]
@razonyang Thank you, ..super quick  :)
